tests/mocks/circular_dependencies_ttl.py

```diff
@@ -27,6 +29,10 @@
         <http://uri.com#String>,
         <http://uri.com#Subject> .
 
+<http://publications.europa.eu/resource/authority/file-type/JSON> a foaf:Document ;
+    dct:format "http://publications.europa.eu/resource/authority/file-type/JSON"^^dct:MediaType ;
```

Comment:
Hvorfor ikke bare bruke URI syntaks her framfor `^^dct:MediaType`?

Reply:
`foaf:Document` har ikke en property som heter `dct:format` (http://xmlns.com/foaf/spec/), så her synes jeg vi skal stille spørsmål til spesifikasjonen

Reply:
Hvis jeg forstår dette korrekt så er det en utvidelse i modelldcat at foaf:Document har dct:format
https://informasjonsforvaltning.github.io/modelldcat-ap-no/#Skal-brukes-for-format

Reply:
Hei, ja det stemmer det som @NilsOveTen sier. @azaxarov det er også derfor det brukes datatype dct:MediaType og ikke vanlig uri. Men det er en URI for alle funksjonelle formål

Reply:
dct:MediaType, som er range til dct:format, er ei klasse. Så her må vi enten
- lage ei blank node (ikkje ønskeleg), eller
- lenke til ein instans av dct:MediaType. (anbefalt)

Så denne skal vere slik:
```
:my_foaf_Document a foaf:Document;
   dct:format <http://publications.europa.eu/resource/authority/file-type/JSON> ;
.
```
Som er i tråd med slik vi gjer det overalt ellers. (Og dette vokabularet ligger/skal ligge i FDK reference data.)